docs/configuration.asciidoc

```diff
@@ -143,14 +154,14 @@ NOTE: The service name must conform to this regular expression: `^[a-zA-Z0-9 _-]
 
 [options="header"]
 |============
-| Environment variable name      | IConfiguration or Web.config key 
+| Environment variable name      | IConfiguration or Web.config key
 | `ELASTIC_APM_SERVICE_NAME` | `ElasticApm:ServiceName`
 |============
 
 [options="header"]
 |============
 | Default                          | Type
-| Name of the entry assembly       | String 
```

Comment:
Sorry about all of these. Just updated my formatting rules. I can remove them from this PR if you'd like.

Reply:
No worries, all valid changes - feel free to keep those.